utils/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -85,6 +94,17 @@ fun String.isSpdxExpressionOrNotPresent(strictness: Strictness = Strictness.ALLO
 fun String.toSpdx(strictness: Strictness = Strictness.ALLOW_ANY): SpdxExpression =
     SpdxExpression.parse(this, strictness)
 
+/**
+ * Parses the string as an [SpdxExpression] of the given [strictness] and returns the result on success, or null if the
+ * string cannot be parsed.
+ */
+fun String.toSpdxOrNull(strictness: Strictness = Strictness.ALLOW_ANY): SpdxExpression? =
+    runCatching {
+        toSpdx(strictness)
+    }.onFailure {
+        logger.debug { "Could not parse license '$this' to SPDX: ${it.collectMessages()}" }
```

Comment:
As $this is not necessary a license, if it could not be parse, should we just drop the mention of "license" from the log message?

Reply:
I've reworded the message.